health-check/src/uyuni_health_check/loki/loki_manager.py

```diff
@@ -143,7 +143,7 @@ def check_series_in_loki(
     start = end - 60 * 60
 
     response = requests.get(
-        f"{loki_url}/loki/api/v1/query",
+        f"{loki_url}/loki/api/v1/query_range",
```

Comment:
Actually looking at the `check_series_in_loki` fn, I see this def:


```
def check_series_in_loki(
    loki_url,
    job_name="promtail-complete-job",
    flag="complete",
    message="Promtail finished!d",
):
```

Shouldn't the `message` be changed as well, since you modified the message?

```
"values": [[str(int(time.time() * 1e9)), "Promtail finished!"]],
```